Code/GeorgiaLibrarySystem-/GTLService/Controller/IController/IMaterialService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using System.ServiceModel;
+using Core;
+
+namespace GTLService.Controller
+{
+	
+	[ServiceContract]
+	public interface IMaterialService
+	{
+        
+        [OperationContract]
+        List<readAllMaterial> GetMaterials(string materialTitle, string author, int numOfRecords = 10, int isbn = 0, string jobStatus = "0");
```

Comment:
is this for showing number of books as well, becouse if it does you have to put realy big number to be sure you see them all? what is jobStatus?

Reply:
you can check the UI, it does show how many books are available for booking right now

Reply:
ill check but what about jobStatus?